dist/index.js

```diff
@@ -29247,12 +29256,15 @@ async function run() {
     try {
         const workflowId = core.getInput('workflowId');
         const chassyRefreshTokenB64 = process.env.CHASSY_TOKEN;
-        if (!chassyRefreshTokenB64) {
+        if (chassyRefreshTokenB64 === undefined) {
             throw new Error('CHASSY_TOKEN not provided in environment');
         }
+        else if (chassyRefreshTokenB64 === '') {
+            throw new Error('CHASSY_TOKEN value is empty string');
+        }
```

Comment:
why do we need the check for an empty string?
how is it different from "CHASSY_TOKEN not provided in environment"?